packages/components/src/components/attribution/overlay.tsx

```diff
@@ -26,7 +26,15 @@ const containerCss = css`
     transition: opacity 150ms ease-in;
 `
 
-const AttributionOverlay = ({ gif, isHovered }: { gif: IGif; isHovered: boolean }) => {
+const AttributionOverlay = ({
+    gif,
+    isHovered,
+    onClick,
+}: {
+    gif: IGif
+    isHovered: boolean
+    onClick?: (gif: IGif) => void | undefined
```

Comment:
@nicksantan I don't think you need the `undefined`

Reply:
@nicksantan I think you defined this in the wrong file, don't forget we have `components` and `react-components`. But let's make the change in both packages

Reply:
@giannif gotcha - this change has been made

Reply:
@nicksantan cool. I think you missed a file, still seeing errors in travis, also looks like you need to catch the branch up to master